oioioi/testrun/migrations/0009_testrunprogramsubmission_input_is_zip.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.16 on 2022-12-20 19:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
is data migration needed? if not I suggest to add some comment in the code why it can be skipped.